### PR TITLE
CI: Add Fedora in build matrix

### PR DESCRIPTION
**Pull request by Biswa96**
Fedora builds android-tools package using this repository. https://src.fedoraproject.org/rpms/android-tools

### CODE REVIEW

**Comment by Biswa96** [comment]
Fedora rawhide has same compiler error as in Opensuse tumbleweed. Rawhide can be uncommented when the compiler error is fixed.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -61,6 +61,10 @@ jobs:
           - os: "alpine"
             pkgs: "gcc"
             env1: "CC=gcc CXX=g++"
+          - os: "fedora:latest"
+            pkgs: "gcc gcc-g++"
+          # - os: "fedora:rawhide"
```
Comment (anatol) [inline .github/workflows/build.yml:66]: do we need this comment?

Comment (Biswa96) [reply .github/workflows/build.yml:66]: I just added it as a reminder and if anyone wanted to test it. It can be enabled in future.

Comment (anatol) [reply .github/workflows/build.yml:66]: it should be either added to the matrix or removed from the config. There is no need to leave the dead code here.

**Comment by anatol** [comment]
> Fedora rawhide has same compiler error as in Opensuse tumbleweed. Rawhide can be uncommented when the compiler error is fixed.

Just saw your comment here.

Add the rawhide. I will take care of the compilation error in a couple of days.

**Comment by anatol** [comment]
Thank you!

**Pull request merged**